src/FunctionApps/python/IntakePipeline/conversion.py

```diff
@@ -187,10 +187,12 @@ def convert_message_to_fhir(
             # ; If an exception occurs while parsing FHIR JSON,
             # Log the full response content
             decoded_response = response.content.decode("utf-8")
-            error_info = f"HTTP Code: {response.status_code}, "\
-            + f"Response Content {decoded_response}"
+            error_info = (
+                f"HTTP Code: {response.status_code}, "
+                + f"Response Content {decoded_response}"
+            )
 
-        logging.error(f"Error during $convert-data -- {error_info}")
+        logging.exception(f"Error during $convert-data -- {error_info}")
```

Comment:
I could be wrong, but I think `logging.exception` is really only for `except` blocks